src/TestSuite/IntegrationTestTrait.php

```diff
@@ -566,10 +559,10 @@ public function controllerSpy(EventInterface $event, ?Controller $controller = n
      * This method will attempt to use the configured exception renderer.
      * If that class does not exist, the built-in renderer will be used.
      *
-     * @param \Throwable $exception Exception to handle.
+     * @param \Throwable|\LogicException|\Cake\Database\Exception|\PHPUnit\Framework\Error\Error $exception Exception to handle.
```

Comment:
Aren't `\LogicException|\Cake\Database\Exception|\PHPUnit\Framework\Error\Error` all extending Throwable?

Reply:
Yes. Is it more ideal to remove them all and use the inheritable Throwable instead?

Reply:
I don't see the point in outlining them all here, if they are subclasses of it.